server/src/main/java/io/spine/server/aggregate/Aggregate.java

```diff
@@ -122,6 +124,8 @@
                                 B extends ValidatingBuilder<S, ? extends Message.Builder>>
                 extends CommandHandlingEntity<I, S, B> {
 
+    private static final Empty EMPTY = Empty.getDefaultInstance();
```

Comment:
Why this is here? It does not belong to an `Aggregate`. Can we please have a nested class for filtering? It should be a singleton. In this way this constant will go there.